src/plugins/data/common/constants.ts

```diff
@@ -108,4 +108,5 @@ export const UI_SETTINGS = {
   QUERY_DATAFRAME_HYDRATION_STRATEGY: 'query:dataframe:hydrationStrategy',
   SEARCH_QUERY_LANGUAGE_BLOCKLIST: 'search:queryLanguageBlocklist',
   NEW_HOME_PAGE: 'home:useNewHomePage',
+  DATA_WITH_LONG_NUMERALS: 'data:withLongNumerals',
```

Comment:
nit: i think i originally did `data` as well. but then was convinced to use `search` because then we can group it with the search category. not a strong opinion on this though